src/main/resources/the_warlordResources/localization/eng/WarlordMod-Card-Strings.json

```diff
@@ -223,8 +222,8 @@
   },
   "the_warlord:DecisiveFactor": {
     "NAME": "Decisive Factor",
-    "DESCRIPTION": "Upgrade all cards in your hand for the rest of combat.",
-    "UPGRADE_DESCRIPTION": "Draw !M! card. NL Upgrade all cards in your hand for the rest of combat."
+    "DESCRIPTION": "Upgrade ALL cards in your hand for the rest of combat.",
+    "UPGRADE_DESCRIPTION": "Upgrade ALL cards in your deck, and ALL cards in your parry deck for the rest of combat."
```

Comment:
The use of the word 'deck' here is not correct in the context of combat. 
I would just use 'Upgrade ALL cards for the rest of combat' for this. Maybe with '(including your Parry cards)' if you feel that would be unclear. Also, the upgraded one should certainly Purge.